### DISCUSSION TITLE
How to convert/transfer WhatsApp Stickers to Signal?

### DISCUSSION THREAD

**Discussion by fredsocial** [signalstickers/signalstickers#581] [Q&A] [answered]
Hi 

Do you know how to convert/transfer WhatsApp Stickers to Signal? There is a bot for Telegram, but I haven't seen anything for Whatsapp

**Comment by romainricard** [comment, marked as answer]
Hello! I wrote a guide for this purpose: https://www.reddit.com/r/signal/comments/kxa631/a_comprehensive_guide_to_find_and_create_stickers/

**Comment by Meteor0id** [comment]
There is now this tool:
https://github.com/laggykiller/sticker-convert

It requires manually copying the sticker forlder from whatsapp but other than that it's automated and fast, with plenty of options to modify and fix the pack before uploading it. Note for example that Whatsapp does not associated emoji to stickers, while Signal requires them. If you simply convert one to the emoji suggestions in Signal will not make much sense.
